### PR TITLE
Feature/employees refactoring (code review)

### PR DESCRIPTION
**Pull request by lukaselmer**

### CODE REVIEW

**Comment by sihu** [comment]
very nice refactoring. Like it 👍  LGTM!

File: src/documents/news/2016-09-01-lunch-presentation.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+date: 2016-09-01
+image_path: /images/news/2016-09-01-lunch-presentation-katas/lunch-presentation-motivation.jpg
+image_gallery_root: /images/news/2016-09-01-lunch-presentation-katas
+image_gallery: ['lunch-presentation-start.jpg', 'lunch-presentation-code1.jpg', 'lunch-presentation-code2.jpg']
+layout: news
+tags: ['news', 'events', 'Knowledge']
+---
+
+<p>To stay up-to-date, improve our knowledge and do it efficiently, we have a format called
+"Lunch Presentations". There Renuo sponsors the food, and someone is presenting while the others are eating.</p>
```
Comment (sihu) [inline src/documents/news/2016-09-01-lunch-presentation.html]: mainly listening while eating?

Comment (lukaselmer) [inline src/documents/news/2016-09-01-lunch-presentation.html]: 👍

Comment (lukaselmer) [inline src/documents/news/2016-09-01-lunch-presentation.html]: https://github.com/renuo/beta.renuo.ch/commit/4fa96ca03320cd51f3119a1ccde1b38f04768e42 ;)

**Pull request merged**